crates/topos-config/src/lib.rs

```diff
@@ -77,7 +70,7 @@ pub(crate) fn load_config<T: Config>(node_path: &Path, command: Option<NodeComma
     }
 }
 
-pub(crate) fn insert_into_toml<T: Config>(config_toml: &mut toml::Table, config: T) {
+pub fn insert_into_toml<T: Config>(config_toml: &mut toml::Table, config: T) {
```

Comment:
Why not use a `From` impl to do this?

Reply:
Its not new code, so I didn't touch the design.
But feel free to suggest code if you want.

Reply:
Thing is: you're making this method public, which means we have to support it (or bear the burden of breaking APIs), so that marks a transition when things like bad names start to matter a bit more.

Reply:
If using `From` is annoying here, consider removing the method and just copy the flattening stuff into where it's used.

Reply:
@dvdplm I don't see how you could implement `From` in this situation. I will see if I can remove or refactor this.